site/public/js/notebook_builder/widgets/widget.js

```diff
@@ -1,6 +1,6 @@
 /* exported NUM_MARKDOWN, Widget */
-const NUM_MARKDOWN = 0;
-
+// eslint-disable-next-line prefer-const
+let NUM_MARKDOWN = 0;
```

Comment:
```suggestion
// eslint-disable-next-line no-unused-vars
const NUM_MARKDOWN = 0;
```
According to what you described on PR, `NUM_MARKDOWN` variable is used globally. However, you are suppressing to use `let` instead of `const`?

Reply:
https://github.com/Submitty/Submitty/blob/9ca03854abffa3cebea8420b25d88f8c2a4f6010/site/public/js/notebook_builder/widgets/markdown-widget.js#L58

oh i missed one point, we are changing this variable `NUM_MARKDOWN `value